packages/ui/src/components/va-button-toggle/VaButtonToggle.vue

```diff
@@ -83,13 +84,15 @@ export default defineComponent({
 
     const { textColorComputed: activeButtonTextColor } = useTextColor(activeButtonColor)
 
+    const isToggled = (value: any) => value === props.modelValue
```

Comment:
also you can do below
`const getButtonClass = (buttonValue: any) => ({ 'va-button--active': isToggled(buttonValue) })`